docs/configuration.md

```
@@ -70,6 +83,27 @@ via `python-dotenv`.
+stored key for a single run without touching the store:
+
+```bash
+GROQ_API_KEY="a_different_key" repo2readme run --local .
+```
+
+## Running without a terminal
+
+In CI, or any time stdin is not a terminal, there is nobody to answer a prompt.
+A missing key is reported straight away instead:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)